api/src/main/java/org/openmrs/PatientProgram.java

```diff
@@ -51,7 +51,7 @@
 	
 	private Set<PatientState> states = new HashSet<>();
          
-        private Set<PatientProgramAttribute> attributes = new LinkedHashSet();
```

Comment:
Did you accidentally format (TAB) the above?

Reply:
@dkayiwa You're right, I didn't notice it. It definitely needs to be corrected.